### ISSUE TITLE
Link similar dataset

### ISSUE THREAD

**Issue by Hoedic** [boxkite/ckanext-donneesqctheme#12]
This kind of portal will probably end up with lots of data on the different subjects from different jurisdictions. For example transit schedule for all the transit agencies of the participating cities. I would be neat to link them together. I was something like a tag vocabulary with predefined values.

**Comment by deniszgonjanin** [comment]
I agree tag vocabulary is probably best for this. @wardi might have a different idea
